File: CRUD-Operations-Angular.DataAccess/ViewModels/UsersProjectsViewModel.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CRUD_Operations_Angular.DataAccess.ViewModels
{
    public class UsersProjectsViewModel: IViewModel
    {
        public int ProjectId { get; set; }
        public int UserId { get; set; }
    }
}

File: CRUD-Operations-Angular/ClientApp/app/components/models/usersProjectsViewModel.ts
export class UsersProjectsViewModel
{
    public userId: number;
    public projectId: number;

    public static GetUserProjectModel(userId: number, projectId: number): UsersProjectsViewModel
    {
        let result: UsersProjectsViewModel = new UsersProjectsViewModel();
        result.projectId = projectId;
        result.userId = userId;
        return result;
    }
}